distribution/windows/setup/radarr.iss

```diff
@@ -34,9 +34,10 @@ DisableReadyPage=True
 CompressionThreads=2
 Compression=lzma2/normal
 AppContact={#ForumsURL}
-VersionInfoVersion={#BaseVersion}.{#BuildNumber}
+VersionInfoVersion={#BaseVersion}
 SetupLogging=yes
 OutputDir=output
+AppverName={#AppName}
```

Comment:
This will show up as `Radarr` instead of `Radarr 4`

Reply:
between this PR / Prowlarr / and upstream....what do we want.